go/vt/topo/etcd2topo/server.go

```diff
@@ -87,6 +128,12 @@ func NewServer(serverAddr, root string) (*Server, error) {
 	}, nil
 }
 
+// TODO: Rename this to a name to signifies this is a global etcd server.
```

Comment:
The term "global" here is overloaded. This function is used for both global and cell-local etcd servers, in the Vitess topo sense of the word "global". I guess what you mean here is that this function uses the process-wide TLS settings. Let's reword the comment to avoid using the word "global".

Reply:
Done https://github.com/vitessio/vitess/pull/5496/commits/894827a69324e42024b35e15f28f8feaf2a9d2f4.